Code/Framework/AzCore/AzCore/IO/Streamer/Statistics.cpp

```diff
@@ -140,9 +140,9 @@ namespace AZ::IO
         [[maybe_unused]] double value)
     {
         AZ_PROFILE_DATAPOINT(AzCore, value,
-            "Streamer/%.*s/%.*s (Raw)",
+            AZStd::string::format(L"Streamer/%.*s/%.*s (Raw)",
```

Comment:
This should be `AZStd::wstring::format`

Reply:
Good callout, I sort of assumed that there was a `wchar_t const*` overload or something. Is it worth a static assert in `AZStd::string::format` to make sure this error isn't made do you think?

Reply:
good point, thanks! fixed

Reply:
A static_assert could be extra protection for the use to make sure they want are using the `AZStd::wstring` name when to make sure it is known that the function will return a **wstring** and not a  _string_